spec/components/jobseekers/schools_overview_component_spec.rb

```diff
@@ -103,10 +110,45 @@
   end
 
   it 'renders all the school links to the school website' do
-    [school_2, school_3].each { |school| expect(rendered_component).to include(school.url) }
+    [school_1, school_2, school_3].each { |school| expect(rendered_component).to include(school.url) }
```

Comment:
This should fail

Reply:
Why? All the schools have a url. In the spec they get it from the from the schools factory, and in our app they get it from GIAS if there isn't one set by the user.